test/ffprober_test.rb

```diff
@@ -2,31 +2,38 @@
 
 class FfproberTest < Minitest::Test
   def test_json_input
-    ffprobe = Ffprober::Parser.from_json(File.read("test/assets/sample video.json"))
+    ffprobe = Ffprober::Parser.from_json(
+      File.read('test/assets/sample video.json')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.